metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -442,11 +443,12 @@
           (trace "Executing" stmt "with values" (pr-str values))
           (when (not (= "efs-only" (efs-config/efs-toggle)))
             (info "Creating Oracle record with efs-toggle value " (efs-config/efs-toggle))
-            (j/db-do-prepared db stmt values)
-            (after-save conn provider concept))
+            (info "Time taken for Oracle insertion: " (first (util/time-execution
+                                                              (j/db-do-prepared db stmt values)
+                                                              (after-save conn provider concept)))) " ms")
```

Comment:
Also the after-save does other things then saving and should also be done if efs only option is being used. 
So you may want to re-write this section to something like

```
(trace "Executing" stmt "with values" (pr-str values))
(when (not (= "efs-only" (efs-config/efs-toggle)))
    (info "Creating Oracle record with efs-toggle value " (efs-config/efs-toggle))
    (info "Time taken for Oracle insertion: " (first (util/time-execution (j/db-do-prepared db stmt values))) " ms"))

(when  (not (= "efs-off" (efs-config/efs-toggle)))
    (info "Creating EFS record with efs-toggle value " (efs-config/efs-toggle))
    (info "Time taken for EFS insertion: " (first (util/time-execution (efs/save-concept provider concept-type (zipmap (map keyword cols) values)))) " ms"))
(after-save conn provider concept)
```

Also I added an extra ) at the end of the first while, because I think you are missing it.  Then you will have an extra one somewhere farther down.